src/cpp/session/modules/SessionGit.cpp

```diff
@@ -127,7 +127,7 @@ core::system::ProcessOptions procOptions()
    // (note that we also do this on init, but we do this again for
    // child processes just to ensure any user-initiated PATH munging
    // doesn't break builtin utilities)
-   FilePath postbackDir = session::options().rpostbackPath().getParent();
+   FilePath postbackDir = module_context::rPostbackScriptsDir();
    core::system::addToPath(&childEnv, postbackDir.getAbsolutePath());
```

Comment:
again like why are we sometimes using `core::system::addToPath` and other times using the new `module_context::appendToPath`?

Reply:
`core::system::addToPath()` should be safe to use when it's modifying an existing C++ object as opposed to the current environment. I'm not sure if there's a good way of enforcing this in the code base though :-/